ui/v2/src/components/Settings/SettingsConfigurationPanel.tsx

```diff
@@ -148,6 +174,26 @@ export const SettingsConfigurationPanel: FunctionComponent<IProps> = (props: IPr
         >
           <InputGroup value={generatedPath} onChange={(e: any) => setGeneratedPath(e.target.value)} />
         </FormGroup>
+
+        <FormGroup
+          label="Excluded Patterns"
+          helperText="Regexps of files/paths to exclude from Scan"
+
+        >
+
+       { (excludes) ? excludes.map((regexp, i) => {
+                                            return(
+                                              <InputGroup
+                                              value={regexp}
+                                              onChange={(e: any) => excludeRegexChanged(i, e.target.value)}
+                                              rightElement={<Button icon="minus" intent="danger" onClick={(e: any) => excludeRemoveRegex(i)} />}
+                                              />
+                                              );
+                                            })   : null
```

Comment:
This should be indented by 2 spaces from line 184. Remove extra two spaces between `)` and `: null`.